packages/cli/src/commands/dev.ts

```diff
@@ -1,7 +1,78 @@
 import { Command } from '@oclif/core'
+import { readFileSync } from 'fs'
+import { Readable } from 'stream'
+import { resolve as resolvePath, sep } from 'path'
+import chokidar from 'chokidar'
+
+import { getRoot } from '../utils/root'
+
+export interface ChangeToCopy {
+  path: string | null
+  content: string | Readable | Buffer | NodeJS.ReadableStream
+}
+
+const stabilityThreshold = process.platform === 'darwin' ? 100 : 200
+
+const defaultPatterns = ['*/**', '**']
+
+const defaultIgnored = [
```

Comment:
why not using .gitignore content in here? Also, we why having specific case for cypress in here?

Reply:
I don't think we want to tie this ignored list to the ones in .gitignore. I feel this may turn into a limitation in the future, like a generated code that we want to trigger the dev command but don't want to include it in git.

The specific case for cypress was present in the toolbelt link command from which I copied portions of this code, probably because they didn't want to upload huge screenshots or video files. I think we can remove it.